tests/components/blackbird/test_media_player.py

```diff
@@ -10,7 +8,7 @@
 from homeassistant.components.blackbird.media_player import (
     DATA_BLACKBIRD,
     PLATFORM_SCHEMA,
-    setup_platform,
+    async_setup_platform,
```

Comment:
I believe if you call `async_setup_component` from `homeassistant.setup` you may be able to revert the changes to the async_setup_platform.
https://developers.home-assistant.io/docs/development_testing/#writing-tests-for-integrations